api/types/events/events.proto

```diff
@@ -1771,6 +1771,7 @@ message OneOf {
         events.SessionConnect SessionConnect = 67;
         events.CertificateCreate CertificateCreate = 68;
         events.SFTP SFTP = 69;
+        events.SessionRecordingAccess SessionRecordingAccess = 70;
```

Comment:
The message number should match the one in master

Reply:
Fixed